### PR TITLE
Implement "Add a class" 

### LINKED ISSUE #7: Add a class
As a TA I can so that I can keep track of my classes.

### PR DESCRIPTION
**Pull request by songgthu**
Done with implementing Adding a class command, here are new changes that I made:

1. New classes added:
- Class
- ClassName
- UniqueClassList
- AddClassCommand
- AddClassCommandParser

2. Update Storage to store the class list (storage will automatically update any changes related to class list)

3. Refractor `AddressBook` to `EduTrack`

### CODE REVIEW

**Review by DonovanJJ — APPROVED** [review]
Changes so far looks great, well documented and unit tested 👍 
Are the updates to Storage still work in progress?

File: src/main/java/seedu/address/logic/parser/CliSyntax.java [hunk 1]
```diff
@@ -12,4 +12,6 @@ public class CliSyntax {
     public static final Prefix PREFIX_ADDRESS = new Prefix("a/");
     public static final Prefix PREFIX_TAG = new Prefix("t/");
 
+    public static final Prefix PREFIX_CLASS = new Prefix("/c");
```
Comment (sudarshan2401) [inline src/main/java/seedu/address/logic/parser/CliSyntax.java:15]: Should we standardise the prefixes to be "/" before the letter or after?

File: src/main/java/seedu/address/model/module/Class.java
Comment (sudarshan2401) [inline src/main/java/seedu/address/model/module/Class.java:1]: LGTM! For now classes only have a class name right?

**Review by sudarshan2401 — APPROVED** [review]
LGTM! Great job with the implementation of the classes and the testing as well!

File: src/main/java/seedu/address/logic/parser/CliSyntax.java (hunk 1 above)
Comment (songgthu) [reply src/main/java/seedu/address/logic/parser/CliSyntax.java:15]: In our user guide we put "/" before so I guess we can keep it that way

**Pull request merged**